backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/controller/SpecificationController.kt

```diff
@@ -56,9 +76,42 @@ class SpecificationController @Autowired constructor(
         return specificationIdFromPath
     }
 
+    @PutMapping("/{specificationId}/chmod/{userId}")
```

Comment:
These are some interesting questions :)
I'd have said that the whole permission thingy should be in the `domain`, because it's not dependent on the interface. On the other hand, the way a user identifies themselves does depend on the interface. So probably some wrapping and converting again and then calling methods in the `domain`?